code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -131,6 +131,7 @@
 	name = "assault shotgun"
 	desc = sawn_desc
 	w_class = WEIGHT_CLASS_NORMAL
+	weapon_weight = WEAPON_MEDIUM
```

Comment:
I wouldn't do this to the riot shotgun; it's really strong too---the main advantage to this should be that you can store it in your backpack.